src/interpreter_funcs/voxelize.rs

```diff
@@ -173,8 +184,8 @@ impl Func for FuncVoxelize {
         log(LogMessage::info(format!("Voxel count = {}", voxel_count)));
 
         if error_if_large && voxel_count > VOXEL_COUNT_THRESHOLD {
-            let vy_over_vx = voxel_dimensions[1] / voxel_dimensions[0];
-            let vz_over_vx = voxel_dimensions[2] / voxel_dimensions[0];
+            let vy_over_vx = voxel_dimensions.y / voxel_dimensions.x;
+            let vz_over_vx = voxel_dimensions.z / voxel_dimensions.x;
```

Comment:
This is the nicer part. 
However, this is a very temporary piece of code because it's being replaced by a more precise method in PR#154